content/blog/2023/04/26/research-community-open-call.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Research Community Open Call #1 – April 2023"
+created: 2023-04-26
+categories:
+  - commmunity-call
+authors:
+  - lifexitself
+---
+
+This is the first in a series of monthly open meetings hosted by the Life Itself research community. These purpose of these calls is to provide an opportunity for newcomers to learn about the community and engage in discussions, and for community members to share updates about ongoing work and coordinate across interest groups.
```

Comment:
Very nice!

Reply:
I would probably link Life Itself research community to `/research` page (using obsidian syntax). Minor point.